packages/Python/lldbsuite/test/lang/swift/parseable_interfaces/shared/TestSwiftInterfaceNoDebugInfo.py

```diff
@@ -48,6 +48,31 @@ def test_swift_interface_fallback(self):
             open(self.getBuildArtifact(module), 'w').close()
         self.do_test()
 
+    @swiftTest
+    @skipUnlessPlatform(["macosx"])
+    def test_prebuilt_cache_location(self):
+        """Verify the prebuilt cache path is correct"""
+        self.build()
+        log = self.getBuildArtifact("types.log")
+        self.runCmd('log enable lldb types -f "%s"' % log)
+
+        # Set a breakpoint in and launch the main executable so we load the
+        # ASTContext and log the prebuilt cache path
+        lldbutil.run_to_source_breakpoint(
+           self, "break here", lldb.SBFileSpec("main.swift"),
+           exe_name=self.getBuildArtifact("main"))
+
+        # Check the prebuilt cache path in the log output
+        found = False
+        prefix = 'Using prebuilt module cache path: '
+        logfile = open(log, "r")
+        for line in logfile:
+            if prefix in line:
+                self.assertTrue(line.endswith('/macosx/prebuilt-modules\n'), 'unexpected prebuilt cache path: ' + line)
+                found = True
+        self.assertTrue(found, 'prebuilt cache path log entry not found')
```

Comment:
would it make sense to also add a check that tests that there is such a directory in `$(xcrun --show-sdk-path)` even if it won't be the same one (because the test is run on a just-built compiler)? This way we could catch the case where the directory is renamed in a future SDK and we need to update the hardcoded path.

Reply:
The directory is in the toolchain, not the SDK…but it also won't exist for a just-built compiler.

Reply:
*Because* it doesn't exist for a just-built compiler, I thought it would be valuable to have an integration test that verifies that the path being constructed here matches the one that the host toolchain also uses.

Reply:
Ah, I see what you mean. That would set a minimum Xcode requirement for building, though. Certainly no released Xcodes have this folder, and we're in the open source repo!

Reply:
@adrian-prantl I filed rdar://problem/50501694 to track adding such this test once public CI is using a new enough Xcode. Are you happy for this to go in in the meantime?